### PR TITLE
FIN-1544: Standard scalafmt settings

### PR DESCRIPTION
**Pull request by mbryzek**
See RFC: https://docs.google.com/document/d/1rPN8FytzkdjgYyp9c1gZAXSZNXqVbmntHo1svnTSV4I/edit#heading=h.m5cxuhapqqb9

### CODE REVIEW

File: api/conf/base.conf [hunk 1]
```diff
@@ -6,9 +6,9 @@ akka.http.parsing.max-uri-length = 16384
 akka.coordinated-shutdown.phases.service-requests-done.timeout = 60 seconds
 
 db.default.driver = "org.postgresql.Driver"
-db.default.hikaricp.maximumPoolSize = 5
+db.default.hikaricp.maximumPoolSize = 10
```
Comment (mchimirev) [inline api/conf/base.conf:9]: this seems unrelated to scalafmt

**Review by mchimirev — APPROVED** [review]

File: api/conf/base.conf [hunk 2]
```diff
@@ -6,9 +6,9 @@ akka.http.parsing.max-uri-length = 16384
...
 db.default.hikaricp.maximumPoolSize = ${?CONF_DB_HIKARICP_MAXIMUM_POOL_SIZE}
-db.default.hikaricp.minimumIdle = 2
+db.default.hikaricp.minimumIdle = 10
```
Comment (jconnolly) [inline api/conf/base.conf:11]: same here

File: api/conf/base.conf (hunk 1 above)
Comment (mbryzek) [reply api/conf/base.conf:9]: fixed the Tests which were otherwise failing

File: api/conf/base.conf (hunk 2 above)
Comment (mbryzek) [reply api/conf/base.conf:11]: fixed the Tests which were otherwise failing
